dapper/mods/L96s/__init__.py

```diff
@@ -0,0 +1,119 @@
+########################################################################################
+########################################################################################
+"""A 1D emulator of chaotic atmospheric behaviour, including additive noise.
+
+`bib.grudzien2020numerical`
+"""
+
+
+########################################################################################
+########################################################################################
+# Imports
+import numpy as np
+
+from dapper.mods.integration import rk4
+
+from .extras import LPs, d2x_dtdx, dstep_dx
+
+########################################################################################
+########################################################################################
+# Method definitions
+########################################################################################
+########################################################################################
+# basic Lorenz 96 defintions and functions
+
+__pdoc__ = {"demo": False}
+
+# energy injected into the system
+Force = 8.0
+
+# diffusion in the SDE process
+s = 0.05
+
+# ?
```

Comment:
It's a suggested window length (duration) for time-series plots.

Reply:
Ah OK, that's helpful.